community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -12,63 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install Ramble
+- name: Install Application
   hosts: localhost
   vars:
+    app_name: ${app_name}
+    profile_script: ${profile_script}
     install_dir: ${install_dir}
-    ramble_url: ${ramble_url}
-    ramble_ref: ${ramble_ref}
+    git_url: ${git_url}
+    git_ref: ${git_ref}
     chmod_mode: ${chmod_mode}
```

Comment:
I suspect that the empty strings here render as "absolutely nothing". In which case, I'm not sure the comparisons to the empy string below are doing what you think they are doing.

The Ansible way is not to compare to empty string anyhow. See comment below.

Reply:
You are more or less correct. It renders as follows:

```
    chmod_mode: a+rwxs
    chown_owner: 
    chgrp_group: 
```

I have tested this with the new comparison that you suggested and it is now working as expected.